custom_dc/README.md

```diff
@@ -125,6 +128,7 @@ In the root of this repository, run:
 
 ```bash
 docker run -it \
+--pull=always \
```

Comment:
I think pulling when running might sometimes lead to unintended surprises.

Perhaps add a section that guides them to pull the latest image explicitly from the repo: `docker pull gcr.io/datcom-ci/datacommons-website-compose:latest`?

Reply:
Why would that give unintended surprises? If we always want users to get refreshed image, then might just have one command to do it?

Reply:
When they run it the first time and they get the refreshed image, that's perfectly ok.

I'm concerned about when they are just bouncing their servers and it pulls in a new image because of `--pull-always`. The new image might have changes / issues they were not expecting.

Reply:
Currently, after rebuilding local docker image, when re-run it, the new docker image is not picked up due to caching. I think that's what we see from Alan's issue.

Reply:
If we really want users to stick with a particular docker image, then shall use <TAG> instead of "latest" or "stable" throughout the documentation. Now "stable" does not match the "stable" version is surprising.

Reply:
Ah, this is for the locally built image and not the one from the repo.

OK - giving lgtm.

We should do a separate discussion on whether moving forward we want customers to build their own image at all or always use ones from our repo. Basically, the service (image) comes from us and the env, data and any webpage customizations from them.